test/test_env_setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
I would prefer to use `sh`, `bash` is not installed in every environment and the script does not use any Bash specific syntax.